confirm/confirm.go

```diff
@@ -130,17 +132,21 @@ func (c *Confirm) afterRegister(ctx *authboss.Context) error {
 		return err
 	}
 
-	goConfirmEmail(c, email, base64.URLEncoding.EncodeToString(token))
+	goConfirmEmail(c, ctx, email, base64.URLEncoding.EncodeToString(token))
 
 	return nil
 }
 
-var goConfirmEmail = func(c *Confirm, to, token string) {
-	go c.confirmEmail(to, token)
+var goConfirmEmail = func(c *Confirm, ctx *authboss.Context, to, token string) {
+	if ctx.DisableGoroutines {
+		c.confirmEmail(ctx, to, token)
+	} else {
+		go c.confirmEmail(ctx, to, token)
+	}
 }
 
 // confirmEmail sends a confirmation e-mail.
-func (c *Confirm) confirmEmail(to, token string) {
+func (c *Confirm) confirmEmail(ctx *authboss.Context, to, token string) {
```

Comment:
This addition causes the tests to fail as it's changed the API. Please run tests on all pull requests. Ideally we'd have CI for this but it never got set up.